py3status/modules/file_status.py

```diff
@@ -68,12 +65,18 @@ def file_status(self):
                 'cached_until': self.py3.CACHE_FOREVER,
             }
 
-        if exists(self.path):
-            icon = self.icon_available
-            color = self.py3.COLOR_GOOD
-        else:
-            icon = self.icon_unavailable
-            color = self.py3.COLOR_BAD
+        paths = glob(expanduser(self.path))
+        if isinstance(paths, str):
+            paths = [paths]
+
+        icon = self.icon_unavailable
+        color = self.py3.COLOR_BAD
+
+        for path in paths:
+            if exists(path):
```

Comment:
If we use `glob`, then we might not need `exists` anymore -- Try `if paths:` instead.

Reply:
Ok for this :)